templates/base.html

```diff
@@ -68,6 +69,19 @@
 <link href="https://fonts.googleapis.com/css?family=Montserrat" rel="stylesheet">
 
 <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.9.1/jquery.min.js"></script>
+<script src="{%static 'js/manup.js' %}"></script>
+<script>
+  if (navigator.serviceWorker.controller) {
+  console.log('[PWA Builder] active service worker found, no need to register')
```

Comment:
If the browser doesn't have support for `serviceWorker`, this line may break JS parsing, by throwing error, `can't read controller of undefined`?

Reply:
Yes it would. Better to have it as ```
if(navigator.serviceWorker && navigator.serviceWorker.controller)```